--- a/vendor/github.com/go-gl/glfw/v3.3/glfw/build.go
+++ b/vendor/github.com/go-gl/glfw/v3.3/glfw/build.go
@@ -26,17 +26,19 @@
 // Linux Build Tags
 // ----------------
 // GLFW Options:
-#cgo linux,!wayland CFLAGS: -D_GLFW_X11
-#cgo linux,wayland CFLAGS: -D_GLFW_WAYLAND -D_GNU_SOURCE
+#cgo linux,!wayland,!combined CFLAGS: -D_GLFW_X11
+#cgo linux,wayland,!combined CFLAGS: -D_GLFW_WAYLAND -D_GNU_SOURCE
+#cgo linux,combined CFLAGS: -D_GLFW_X11
 
 // Linker Options:
 #cgo linux,!gles1,!gles2,!gles3,!vulkan LDFLAGS: -lGL
 #cgo linux,gles1 LDFLAGS: -lGLESv1
 #cgo linux,gles2 LDFLAGS: -lGLESv2
 #cgo linux,gles3 LDFLAGS: -lGLESv3
 #cgo linux,vulkan LDFLAGS: -lvulkan
-#cgo linux,!wayland LDFLAGS: -lX11 -lXrandr -lXxf86vm -lXi -lXcursor -lm -lXinerama -ldl -lrt
-#cgo linux,wayland LDFLAGS: -lwayland-client -lwayland-cursor -lwayland-egl -lxkbcommon -lm -ldl -lrt
+#cgo linux,!wayland,!combined LDFLAGS: -lX11 -lXrandr -lXxf86vm -lXi -lXcursor -lm -lXinerama -ldl -lrt
+#cgo linux,wayland,!combined LDFLAGS: -lwayland-client -lwayland-cursor -lwayland-egl -lxkbcommon -lm -ldl -lrt
+#cgo linux,combined LDFLAGS: -lX11 -lXrandr -lXxf86vm -lXi -lXcursor -lm -lXinerama -ldl -lrt
 
 // BSD Build Tags
 // ----------------
